src/vm/arm64/stubs.cpp

```diff
@@ -1246,7 +1246,8 @@ void UMEntryThunkCode::Encode(BYTE* pTargetCode, void* pvSecretParam)
 
 void UMEntryThunkCode::Poison()
 {
-
+    // Insert 'brk 0xbe' at the entry point
+    m_code[0] = 0xd42017c0;
```

Comment:
@parjong It is not clear to me why none of the `Poison()` functions call `FlushInstructionCache()`.  I assume you know it is not necessary.